src/Endpoints/Delegates/HandlesSettings.php

```diff
@@ -102,6 +102,29 @@ public function resetDisplayedAttributes(): array
         return $this->http->delete(self::PATH.'/'.$this->uid.'/settings/displayed-attributes');
     }
 
+    // Settings - Localized attributes
+
+    /**
+     * @return array{ locales: list<non-empty-string>, attributesPattern: string }
+     */
+    public function getLocalizedAttributes(): ?array
+    {
+        return $this->http->get(self::PATH.'/'.$this->uid.'/settings/localized-attributes');
+    }
+
+    /**
+     * @param array{ locales: list<non-empty-string>, attributesPattern: string } $localizedAttributes
```

Comment:
```suggestion
     * @param array{locales: list<non-empty-string>, attributesPattern: string} $localizedAttributes
```

Reply:
looking at the notion site I see `attributePattern` and not `attributesPattern` ?

Reply:
Hey, I double-checked the code and PRD.
There was a typo _in my PR_ and in the PRD.
The real name is `attributePatterns`.

> attributesPattern: non-empty-string} $localizedAttributes

Here, the `attributesPattern` can be an empty string, so it should just be `string`, right?

Reply:
And what an empty string does represent?

Reply:
I guess we can see that as an exact match on an empty string hehe